src/node_options.cc

```diff
@@ -1009,6 +1010,68 @@ void Initialize(Local<Object> target,
 }
 
 }  // namespace options_parser
+
+void HandleEnvOptions(std::shared_ptr<EnvironmentOptions> env_options) {
+  HandleEnvOptions(env_options, [](const char* name) {
+    std::string text;
+    return credentials::SafeGetenv(name, &text) ? text : "";
+  });
+}
+
+void HandleEnvOptions(std::shared_ptr<EnvironmentOptions> env_options,
+                      std::function<std::string(const char*)> opt_getter) {
```

Comment:
`std::function<std::string(const char*)> opt_getter` is probably not the simplest solution here.

Reply:
I think it’s just fine if you’d like to avoid using a template (which is reasonable, imo)